docs/development/testguide.rst

```diff
@@ -987,73 +966,3 @@ from `astropy-benchmarks <https://github.com/astropy/astropy-benchmarks/>`_.
 It is important to note that these benchmarks can be flaky as they run on
 virtual machines (and thus shared hardware) but they should give a general
 idea of the performance impact of a pull request.
-
-.. _pytest-plugins:
-
-Pytest Plugins
-**************
-
-The following ``pytest`` plugins are maintained and used by Astropy. They are
```

Comment:
We do maintain `pytest-remotedata`, but we transferred `pytest-doctestplus` to Scientific Python in June 2023.